llmfoundry/models/layers/attention.py

```diff
@@ -572,7 +572,8 @@ def __init__(
 
     def forward(
         self,
-        x: torch.Tensor,
+        hidden_states: torch.Tensor,
```

Comment:
to avoid breaking existing calls to these functions, can you leave the argument name and ordering (i.e. put the new arg at the end)?